src/components/LoadConfig.vue

```diff
@@ -1,23 +1,43 @@
 <template>
-  <div class="welcome">
-    <h1>Please select starting point!</h1>
-    <n-split direction="horizontal" style="height: 450px" :max="0.75" :min="0.25">
+  <div class="load-config">
+    <h1 class="title">Please select starting point!</h1>
+    <n-split direction="horizontal" class="content-split">
       <template #1>
-        <div :style="{ height: '450px' }">
-          ( TODO: put opemn config icon here )
-          <n-button @click="load_config" type="primary" ghost>
-            Load instalation config.
-          </n-button>
-          <pre v-if="Object.keys(rust_settings).length > 0">{{ JSON.stringify(rust_settings, null, 2) }}</pre>
+        <div class="option-panel">
+          <div class="option-content">
+            <div class="icon-container">
+              <svg class="icon" viewBox="0 0 24 24" fill="none" stroke="currentColor">
+                <path stroke-linecap="round" stroke-linejoin="round" stroke-width="2"
+                  d="M5 19a2 2 0 01-2-2V7a2 2 0 012-2h4l2 2h4a2 2 0 012 2v1M5 19h14a2 2 0 002-2v-5a2 2 0 00-2-2H9a2 2 0 00-2 2v5a2 2 0 01-2 2z" />
+              </svg>
+            </div>
+            <h2 class="option-title">Load Configuration</h2>
+            <p class="option-description">Load an existing configuration file to start the installation process</p>
```

Comment:
Is it possible to implement some drag and drop functionality ? It would be cool to drag and drop the config here to load config files for users. If possible we can add it for future feature request.